pkg/volume/glusterfs/glusterfs.go

```diff
@@ -335,31 +335,38 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	options = append(options, "backup-volfile-servers="+dstrings.Join(addrlist[:], ":"))
 	mountOptions := volume.JoinMountOptions(b.mountOptions, options)
 
-	// Avoid mount storm, pick a host randomly.
-	// Iterate all hosts until mount succeeds.
-	for _, ip := range addrlist {
-		errs = b.mounter.Mount(ip+":"+b.path, dir, "glusterfs", mountOptions)
-		if errs == nil {
-			glog.Infof("glusterfs: successfully mounted %s", dir)
-			return nil
-		}
-
-		const invalidOption = "Invalid option auto_unmount"
-		if dstrings.Contains(errs.Error(), invalidOption) {
-			// Give a try without `auto_unmount` mount option, because
-			// it could be that gluster fuse client is older version and
-			// mount.glusterfs is unaware of `auto_unmount`.
-			noAutoMountOptions := make([]string, len(mountOptions))
-			for _, opt := range mountOptions {
-				if opt != "auto_unmount" {
-					noAutoMountOptions = append(noAutoMountOptions, opt)
-				}
-			}
-			errs = b.mounter.Mount(ip+":"+b.path, dir, "glusterfs", noAutoMountOptions)
+	// with `backup-volfile-servers` mount option in place, it is not required to
```

Comment:
@jsafrane  you are correct, One is enough. However I was thinking give atleast couple more chances in a scenario where there was a network outage which caused the client to not reach any of the servers. In this patch, the default attempt count (indx) has made to "3", may be I can reduce it to "2" . I feel thats enough. As mentioned in commit, its just about giving some more time before mount client declares the mount is failure. I am making it as "2" instead of "3" . Thanks!